src/github/client/github-client.ts

```diff
@@ -76,14 +76,35 @@ export default class GitHubClient {
 	public async getPullRequests(
 		owner: string,
 		repo: string,
-		pageSize: number,
-		page: number): Promise<Octokit.PullsListResponseItem[]> {
+		pageSize?: number,
+		page?: number,
+		options?: any
```

Comment:
I wonder how do we use these options? Would it better to specify a concrete parameters here to remove ambiguity?

Reply:
Yeah, I would very much like to see typed parameters here. Or maybe a type like `GetPullRequestsOptions` with all the options.